pkg/client/unversioned/remotecommand/remotecommand.go

```diff
@@ -98,7 +98,7 @@ func NewExecutor(config *restclient.Config, method string, url *url.URL) (Stream
 // to wrap the round tripper. This method may be used by clients that are lower level than
 // Kubernetes clients or need to provide their own upgrade round tripper.
 func NewStreamExecutor(upgrader httpstream.UpgradeRoundTripper, fn func(http.RoundTripper) http.RoundTripper, method string, url *url.URL) (StreamExecutor, error) {
-	var rt http.RoundTripper = upgrader
```

Comment:
Hrm - the code on the left looks more canonical in terms of our normal casts-to-prove-types than the code on the right.  Would the linter complain at:

```
rt := http.RoundTripper(upgrader)
```

?